build.sbt

```diff
@@ -69,6 +69,12 @@ lazy val commonSettings = Seq(
     // I think we're good with the classpath exception in there.
     case DepModuleInfo("jakarta.transaction", "jakarta.transaction-api", _) => true
   },
+  libraryDependencies ++= Seq(
+    "org.slf4j" % "slf4j-api" % "2.0.13",
+    "org.slf4j" % "slf4j-log4j12" % "2.0.13" % Test,
+    "org.apache.logging.log4j" % "log4j-slf4j2-impl" % "2.23.1",
+    "org.apache.logging.log4j" % "log4j-api" % "2.23.1",
+  ),
```

Comment:
Can we remove the log4j dependencies from [line 182](https://github.com/unitycatalog/unitycatalog/pull/273/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R182) and other places?

Reply:
Also maybe use the log4jVersion variable?

Reply:
PTAL

Reply:
I had added this change at line 31 so I think we can close this PR